### PR TITLE
Grammatical correction

### PR DESCRIPTION
**Pull request by NdagiStanley**
Friedger and the team, this PR and the subsequent ones are my first contributions to GDG wisdom :+1: .
They are minor correction but I must say the document is very comprehensive.

### CODE REVIEW

File: gdg_events/android_study_jam.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 - Google Developers Study Jams is a free series of global, community-run, in-person study groups. 
 - The first Study Jams series started in 2015. The 2016 series officially starts between February 1, 2016 and May 1, 2016, and is presented in partnership with Udacity. 
-- Google Developer Group facilitators -- consisting of members within the GDG community itself and, and have gone through the curriculum prior to the start of the series -- lead the Study Jams that meet at least once or on a weekly basis, between 1 February 2016 to 1 May 2016.
+- Google Developer Group facilitators -- consisting of members within the GDG community itself and have gone through the curriculum prior to the start of the series -- lead the Study Jams that meet at least once a week or rather on a weekly basis, between 1 February 2016 to 1 May 2016.
```
Comment (friedger) [inline gdg_events/android_study_jam.md:8]: Once a week or weekly basis is the same isn't it?

Comment (NdagiStanley) [inline gdg_events/android_study_jam.md:8]: True. I wasn't sure what was the intended meaning before.

Comment (friedger) [inline gdg_events/android_study_jam.md:8]: The study jam could be a one time event or a weekly event serie

Comment (NdagiStanley) [inline gdg_events/android_study_jam.md:8]: Okay

_Stanley Ndagi /_\* Andela.com
http://andela.com/?utm_source=andela&utm_medium=email&utm_campaign=signature *
_Software Engineer / 0718 154504 <+254718154504> / @Andela
http://bit.ly/andelatwitter_

*"The startup that's harder to get into than Harvard" - *CNN
http://snip.ly/HfId
See what The Wall Street Journal http://snip.ly/u9AI has to say.

**Comment by friedger** [comment]
If you do not want to make the changes I will close this PR for now.

**Comment by NdagiStanley** [comment]
I guess it is okay.

**Comment by NdagiStanley** [comment]
Sorry about that. You can close it :-)

**Comment by friedger** [comment]
You can close your PRs as well.

**Pull request closed without merging**